### DISCUSSION TITLE
Allow .jsonl files to be attached to issues

### DISCUSSION THREAD

**Discussion by sethmlarson** [community/community#28668] [Projects and Issues]
`.jsonl` is a text-based format very similar to JSON. This file format recently now has the project [in-toto](https://in-toto.io/) using it for artifact attestations (ie `artifacts.in-toto.jsonl`). Not being able to upload these files to GitHub issues makes it difficult to create reproducers for projects building on in-toto like [SLSA](https://slsa.dev/). Recently I had to rename all the files for bug reports to `.txt` and provide context for this multiple times, was very tedious for me and likely on the person trying to reproduce the bug, too.

Can we add `.jsonl` to the list of file extensions that are allowed to be attached to GitHub issues? (And for that matter, could we extend the list to all text-based formats?)

**Comment by joffrey-bion** [comment]
I have the same issue. Are there any plans to allow this?
